tools/AzPreview/AzPreview.psm1

```diff
@@ -0,0 +1,308 @@
+#
+# Script module for module 'AzPreview' that is executed when 'AzPreview' is imported in a PowerShell session.
```

Comment:
when is this being used?

Reply:
I think this might have been used when generating the nuget package for testing. Powershell team said not to worry about extra files in the final PR though, they said they would remove any unnecessary files